otbenchmark/_SensitivityConvergence.py

```
@@ -193,23 +230,32 @@ def computeSobolSample(
-                    "Elapsed = %.1f (s), Sample size = %d" % (elapsedTime, sample_size)
-                )
+                print(f"Elapsed = {elapsedTime:.1f} (s), Sample size = {sample_size}")
             for j in range(self.numberOfRepetitions):
-                first_order_AE, total_order_AE = self.computeError(
-                    sample_size,
-                )
+                try:
+                    first_order_AE, total_order_AE = self.computeError(
+                        sample_size,
+                    )
+                except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)